### ISSUE TITLE
Variant of agendaupdate doing round-trip through Google Sheets instead of local TSV

### ISSUE THREAD

**Issue by dbarnett** [insanum/gcalcli#775]
Could agendaupdate or a similar command let you modify events via Google Sheets by:

1. Dumping existing events into a new Sheet
2. Waiting for you to edit/add the rows to specify what changes to make on events
3. Once you're ready, taking the data in that sheet as input an performing the given event updates

?

@michaelmhoffman

**Comment by dbarnett** [comment]
Note people who have worked at the big G might be familiar with this flow by the name "bulkupdate". =)

**Comment by michaelmhoffman** [comment]
Yes, this would definitely be possible. I would love to see this! The workflow in (michaelmhoffman/gcaledit)[https://github.com/michaelmhoffman/gcaledit] does this with a desktop spreadsheet application instead, but doing it with Google Sheets would be less clunky and more universal in a number of ways.

Also, `bulkupdate` might be a better name for the `agendaupdate` subcommand. We would probably want to rename the `updates` subcommand to something else.
